### PR TITLE
Add support to send a set of blocks

### PR DESCRIPTION
**Pull request by riasvdv**
This allows you to send a set of [blocks](https://api.slack.com/block-kit) as a Slack message, which allows for nice formatting

### CODE REVIEW

File: src/Jobs/SendToSlackChannelJob.php [hunk 1]
```diff
@@ -22,15 +22,17 @@ class SendToSlackChannelJob implements ShouldQueue
...
+        public ?string $text = null,
+        public ?array $blocks = null,
     ) {
     }
 
     public function handle(): void
     {
-        $payload = ['type' => $this->type, 'text' => $this->text];
+        $payload = $this->text
+            ? ['type' => 'mrkdwn', 'text' => $this->text]
```
Comment (riasvdv) [inline src/Jobs/SendToSlackChannelJob.php:34]: I've removed the `type` parameter as it was hardcoded to `mrkdwn` anyway, I don't think this is breaking?

Comment (freekmurze) [reply src/Jobs/SendToSlackChannelJob.php:34]: That's ok indeed

**Comment by freekmurze** [comment]
Looks good to me! Could you also add an example to the readme?

**Comment by freekmurze** [comment]
Thanks!

**Pull request merged**